WeakAuras/Init.lua

```diff
@@ -36,6 +36,9 @@ end
 WeakAuras.PowerAurasPath = "Interface\\Addons\\WeakAuras\\PowerAurasMedia\\Auras\\"
 WeakAuras.PowerAurasSoundPath = "Interface\\Addons\\WeakAuras\\PowerAurasMedia\\Sounds\\"
 
+-- force enable WeakAurasCompanion because some addon manager interfer with it
+EnableAddOn("WeakAurasCompanion")
```

Comment:
Did you test this without the companion addon installed? I want to be sure that we don’t error out during init for 80%+ of our users :D

Reply:
Also, “interfere”

Reply:
yes `/run EnableAddOn("hellorivers")` doesn't raise any error